react-app/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "react-app",
+  "version": "1.0.0",
+  "description": "React App using Babel, Webpack and SASS",
+  "main": "index.js",
+  "scripts": {
+    "clean": "rimraf ./dist/*",
+    "build": "npm run clean && webpack -p",
+    "start": "npm run clean && webpack-dev-server --hot"
+  },
+  "keywords": [
+    "React"
+  ],
+  "author": "Akshay Pawar",
+  "license": "ISC",
+  "devDependencies": {
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.4",
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-register": "^6.26.0",
+    "css-loader": "^0.28.11",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
```

Comment:
@swapnil-webonise @abhinav-webonise  please suggest a solution here, the stable package itself doesn't provide support for Webpack 4, but provides support in Beta version.
https://github.com/webpack-contrib/extract-text-webpack-plugin/issues/701
However this plugin is featured in Webpack Docs and is widely used. 
https://webpack.js.org/plugins/extract-text-webpack-plugin/
Should we stop using this?

Reply:
I still don't see a concrete solution here. Please suggest how should we solve such a problem where we aren't able to extract css to a file, along with SASS in it. Also there's a high possibility that we might use Webpack 4 in the Iconic rebuild, so this problem is going to arise there also, it's better we solve it here now.

Reply:
use webpack 3 Stable supporting version, until the time they aren't supporting or find an alternative to your beta package.

Reply:
Solved. Used another plugin named `mini-css-extract-plugin`, https://github.com/webpack-contrib/mini-css-extract-plugin. It supports extraction of CSS to a seperate file and also SASS.